vgstation13.dme

```diff
@@ -2291,6 +2291,7 @@
 #include "maps\randomvaults\dance_revolution.dm"
 #include "maps\randomvaults\objects.dm"
 #include "maps\randomvaults\spessmart.dm"
+#include "maps\randomvaults\assistantslair.dmm"
```

Comment:
🤔

Reply:
/shrug

I unticked it and pushed the changes.  My bad.